app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeModel.kt

```diff
@@ -0,0 +1,31 @@
+package org.fossasia.openevent.general.attendees
+
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import com.github.jasminb.jsonapi.IntegerIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Relationship
+import com.github.jasminb.jsonapi.annotations.Type
+import org.fossasia.openevent.general.event.EventId
+import org.fossasia.openevent.general.ticket.TicketId
+
+
+@Type("attendee")
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
+data class AttendeeModel(
+        @Id(IntegerIdHandler::class)
+        var firstName: String? = null,
+        var lastName: String? = null,
```

Comment:
change it to `lastname` and try again. `lastName` -> `lastname`
`firstName` to `firstname`.

Reply:
@srv-twry already did it locally. doesn't work.
I will push the latest code to avoid confusion